bookmarks.py

```diff
@@ -0,0 +1,117 @@
+from peewee import *
+import sqlite3
+from database_config import database_path
+from models import Bookmarks, Cache
+import json
+from pprint import pprint
+
+
+# example_imgur_dictionary =  {'image_url': 'https://www.google.com'}
```

Comment:
All of this commented-out code doesn't belong in a PR. If you need notes, make scratch or temp files, don't add to the main codebase.